vcd/datasource_vcd_catalog_item.go

```diff
@@ -0,0 +1,33 @@
+package vcd
+
+import "github.com/hashicorp/terraform/helper/schema"
+
+func datasourceVcdCatalogItem() *schema.Resource {
+	return &schema.Resource{
+		Read: resourceVcdCatalogItemRead,
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"catalog": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "catalog name where upload the OVA file",
```

Comment:
Description isn't correct

Reply:
Copied from resource.

Reply:
Yes, and in data source it sound strange. 
```suggestion
				Description: "catalog name where the OVA file uploaded",
```

Reply:
fixed